File: Lenguaje de Programación 2/Labs/20151521 (2)/20151521/Especialidad.java
public class Especialidad{
	private String nombre;
	private String clasif;
	
	public Especialidad (String nombre,String clasif){
		this.nombre=nombre;
		this.clasif=clasif;
	}
	
	public void setNombre(String nombre){
		this.nombre=nombre;
	}
	
	public String getNombre(){
		return this.nombre;
	}
	
	public void setClasif(String clasif){
		this.clasif=clasif;
	}
	public String getClasif(){
		return this.clasif;
	}
	
	
}

File: Lenguaje de Programación 2/LP/Parcial/L3/Especialidad.cs
public class Especialidad{
	private string nombre;
	private string clasif;
	
	public Especialidad (string nombre,string clasif){
		this.nombre=nombre;
		this.clasif=clasif;
	}
	
	public void setNombre(string nombre){
		this.nombre=nombre;
	}
	
	public string getNombre(){
		return this.nombre;
	}
	
	public void setClasif(string clasif){
		this.clasif=clasif;
	}
	public string getClasif(){
		return this.clasif;
	}
	
	
}